src/battle_script_commands.c

```diff
@@ -2739,18 +2739,9 @@ void SetMoveEffect(bool32 primary, u32 certain)
                 }
                 break;
             case MOVE_EFFECT_FLINCH:
-                if (GetBattlerAbility(gEffectBattler) == ABILITY_INNER_FOCUS)
+                if (gBattleMons[gEffectBattler].ability == ABILITY_INNER_FOCUS)
```

Comment:
Why was this changed?

Reply:
Honestly, I don't remember why I changed that at all 😂 
I'll put it back to normal and test it out :eyes: